source/vibe/data/json.d

```diff
@@ -181,6 +199,8 @@ struct Json {
 	/// ditto
 	long opAssign(long v) { m_type = Type.int_; m_int = v; return v; }
 	/// ditto
+	BigInt opAssign(BigInt v) { m_type = Type.bigint; initBigInt(); m_bigint = v; return v; }
```

Comment:
This case is fixed.
`initBigInt()` is called only if old `m_type` in assign is not `Type.bigint`